### DISCUSSION TITLE
New Vertical Grids

### DISCUSSION THREAD

**Discussion by fobryan3** [NCAR/omwg_dev#31] [General]
As discussed in 3/29/2024 MOM6 meeting, we are trying to create a new 75 layer z-star grid that can also serve as a source for nominal thicknesses in the z-star region of a hycom1 grid, and possibly as guidance for maximum dz's in the isopycnal part of the hycom1 grid. I generated the grid pictured below using a code that represents the resolution function in three parts: 1) a (near) constant resolution surface layer: 0-20m, 2) a deep layer with an exponential dz profile : below ~200m and 3) a transition between the two with a tanh profile. The added resolution over the current 65 layer z-star is mostly between 10m and 300m. The figure below shows the result in compared to the POP 60 layer grid, the old MOM6 65 layer z-star grid and the current MOM6 75 layer hycom1 grid. I tried eyeballing the resolution function against the modal resolution criteria in Fig 6 of Stewart et al (2017). It looks like this grid should represent modes 1 and 2 pretty well, but would be marginal for mode 3. Comments?
<img width="718" alt="Screenshot 2024-03-29 at 5 07 26 PM" src="https://github.com/NCAR/omwg_dev/assets/47898629/b2b007ea-b0cc-4de5-9933-9c56526ec145">

**Comment by fobryan3** [comment]
For anyone who wants to play with the code/grid you can currently find it in:
/glade/u/home/bryan/MOM6-modeloutputanalysis/VerticalGrid/generate_zstar_grid.ipynb
/glade/u/home/bryan/MOM6-modeloutputanalysis/VerticalGrid/zstar_75layer_2.5m_248.4m-2024-03-29.nc

**Comment by gustavo-marques** [comment]
Thanks for generating this new grid. I will try running a case with it over the weekend.
